src/Native/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ if (APPLE)
     add_custom_command(TARGET System.Security.Cryptography.Native POST_BUILD
         COMMAND ${CMAKE_INSTALL_NAME_TOOL} -change /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib @rpath/libcrypto.1.0.0.dylib $<TARGET_FILE:System.Security.Cryptography.Native>
         COMMAND ${CMAKE_INSTALL_NAME_TOOL} -change /usr/local/opt/openssl/lib/libssl.1.0.0.dylib @rpath/libssl.1.0.0.dylib $<TARGET_FILE:System.Security.Cryptography.Native>
-        COMMAND ${CMAKE_INSTALL_NAME_TOOL} -add_rpath @loader_path $<TARGET_FILE:System.Security.Cryptography.Native>
+        COMMAND ${CMAKE_INSTALL_NAME_TOOL} -add_rpath @loader_path -add_rpath /usr/local/share/dotnet/redist $<TARGET_FILE:System.Security.Cryptography.Native>
```

Comment:
Is that the order we want?  According to https://cmake.org/Wiki/CMake_RPATH_handling it goes:

- DYLD_LIBRARY_PATH - an environment variable which holds a list of directories
- RPATH - a list of directories which is linked into the executable. These can contain @loader_path and @executable_path.
- builtin directories - /lib /usr/lib
- DYLD_FALLBACK_LIBRARY_PATH - an environment variable which holds a list of directories

So specifically for us it will be:

- DYLD_LIBRARY_PATH
- `@loader_path`
- `/usr/local/lib`
- `/usr/local/share/dotnet/redist`
- builtin directories
- DYLD_FALLBACK_LIBRARY_PATH